tutorials/11-probabilistic-pca/11-probabilistic-pca.jmd

```diff
@@ -0,0 +1,570 @@
+---
+title: Probabilistic Principal Component Analysis
+permalink: /:collection/:name/
+---
+
+Principal component analysis is a fundamental technique to analyse and visualise data.
+You may have come across it in many forms and names.
+Here, we give a probabilistic perspective on PCA with some biologically motivated examples.
+For more details and a mathematical derivation, we recommend Bishop's textbook (Christopher M. Bishop, Pattern Recognition and Machine Learning, 2006).
+
+```julia
+using Turing
+using Distributions, LinearAlgebra
+
+# Packages for visualization
+using VegaLite, DataFrames, StatsPlots
+
+# Import Fisher's iris example data set
+using RDatasets
+
+# Set a seed for reproducibility.
+using Random
+Random.seed!(1789);
+```
+
+## A basic example
+
+The idea of PCA is to find a latent variable $z$ that can be used to describe hidden structure in our dataset.
+We use a simple Gaussian prior for
+$$
+P(z) = \mathcal{N}(z | 0, I)
+$$
+and similarly, the conditional distribution
+$$
+P(x | z) = \mathcal{N}(x | W z + \mu, \sigma^2 I)
+$$
+is modeled via a Gaussian distribution.
+
+### A biologically motivated example
+
+We'll generate synthetic data to explore the models.
+The simulation is inspired by biological measurement of
+expression of genes in cells, and so you can think of the two variables as cells and genes.
+While the human genome is (mostly) identical between all the cells in your body, there exist interesting differences in gene expression in different human tissues and disease conditions.
+Similarly, one way to investigate certain diseases is to look at differences in gene expression in cells from patients and healthy controls (usually from the same tissue).
+
+Usually, we can assume that the changes in gene expression only affect a subset of all genes (and these can be linked to diseases in some way).
+One of the challenges of this kind of data is to see the underlying structure, e.g. to make the connection between a certain state (healthy/disease) and gene expression.
+The problem is that the space we are investigating is very large (Up to 20000 genes across 1000s of cells). So in order to find structure in this data, we usually need to project the data into a lower dimensional space.
```

Comment:
This stuff is a good motivation, and might be a good candidate to generalize and move to the top paragraph.